src/test/resources/pages/instructorFeedbackEditCopyFail.html

```diff
@@ -1081,9 +1081,12 @@ <h5 class="col-sm-2">
           </div>
         </div>
         <br>
-        <div class="col-sm-12 margin-bottom-15px padding-15px background-color-light-green">
+        <div class="col-sm-12 margin-bottom-15px padding-15px alert alert-danger">
           <div class="margin-bottom-7px">
-            <b class="visibility-title">
+            <h4>
+              Changing the visibility after collecting responses is not recommended.
+            </h4>
+            <b class="">
```

Comment:
Yep, got that from the Travis Log. Have fixed using god mode in a5d63f2. Thanks! 😀